index.html

```diff
@@ -0,0 +1,708 @@
+                            </div>
+                        </div>
+                        <hr>
+                        <div class="latest-funding-card--news">
+                            <a href="#">
+                                <p>SubjectWell nabs $10 mln Series A <span>on PE Hub</span></p>
+                            </a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="ads-section padded-page-section flex-row">
+                <div class="ad-container">
+                    <h3 class="section-header">Apps ></h3>
+                    <div class="ad-image flex-column ad-image-one">
+                        <div class="ad-image-overlay"></div>
+                        <h5>
+                            MOST POPULAR
+                        </h5>
+                        <a href="#"><h2>Watch Apple's trailer for its show about the Apollo 11 mission, For All Mankind</h2></a>
+                        <ul class="flex-row">
+                            <li><a href="#">Ivan Mehta</a></li>
+                            <li>1 day ago</li>
+                        </ul>
+                    </div>
+                    <div class="ad-latest flex-row">
+                        <div class="ads-latest-text">
+                            <p>LATEST</p>
+                            <h5>Ubisoft reveals its list of games for Uplay+ and I have questions</h5>
+                        </div>
+                        <div class="latest-image"></div>
+                    </div>
+                    <ul class="ads-list">
+                        <li><a href="#">Here's how to mute those Instagram Story spammers</a></li>
+                        <li><a href="#">Google declutters its news tab on desktop search pages with a fresh look</a></li>
+                        <li><a href="#">North Korea launches an ebook to spread ideology amongst its people</a></li>
+                        <li><a href="#">Google's taking another crack at building a social network</a></li>
+                        <li><a href="#">Tinder made lite app that's 25x smaller than the original</a></li>
+                    </ul>
+                </div>
+                <div class="ad-container">
+                    <h3 class="section-header">Gear ></h3>
+                    <div class="ad-image flex-column ad-image-two">
+                        <div class="ad-image-overlay"></div>
+                        <h5>
+                            MOST POPULAR
+                        </h5>
+                        <a href="#"><h2>Watch Apple's trailer for its show about the Apollo 11 mission, For All Mankind</h2></a>
+                        <ul class="flex-row">
+                            <li><a href="#">Ivan Mehta</a></li>
+                            <li>1 day ago</li>
+                        </ul>
+                    </div>
+                    <div class="ad-latest flex-row">
```

Comment:
Side tip: One of Adblock extensions techniques to recognize ads and then block them is by checking class names. Any class that contains `advertisment, ads, ad, etc` will be blocked by adblock, although they are not really ads. So I suggest to change the class names of any sections uses any of these keywords.

Reply:
The ads have been removed, thank you for the information